src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStoreV1.swift

```diff
@@ -648,7 +647,7 @@ class ContactDiaryStoreV1: DiaryStoring, DiaryProviding {
 
 	private let databaseQueue: FMDatabaseQueue
 
-	private func openAndSetup() -> DiaryStoringVoidResult {
+	func openAndSetup() -> DiaryStoringVoidResult {
```

Comment:
really no longer private?

Reply:
Was testing inhering from V1 approach, forgot to change it back, will update the PR soon